.github/workflows/stale.yml

```diff
@@ -0,0 +1,17 @@
+name: 'Close stale issues'
+on:
+  schedule:
+    - cron: '50 5 * * *'
+
+permissions:
+  issues: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v9
+        with:
+          days-before-stale: 30
+          any-of-labels: needs-info
```

Comment:
Is this the most suitable name for the label, or anybody have other suggestions?

Reply:
I think, this is good.